### ISSUE TITLE
Line 91 invalid syntax error

### ISSUE THREAD

**Issue by sarahefcarroll** [richford/dvr_py#1]
When I run this script through my pyzo install I keep getting a syntax error at line 91:
line 91 
print 'The first {n:d} energies are:'.format(n=num_eigs)
                                        ^
SyntaxError: invalid syntax
and I can't find the problem the  comment on the line above is formatted correctly, as is the precision command on line 88. Any ideas?

**Comment by richford** [comment]
@sarahefcarroll, this might just be a python2 vs python3 issue since python3 doesn't accept the old python2 print statement. I really ought to upgrade dvr_py to python3 but in the meantime, does it work for you if you run it under python 2.7?

If you don't already have both a python2 and a python3 environment on your computer and want some guidance, you could check out [this guide](http://www.wilsonsayreslab.org/blog/2015/10/26/managing-multiple-python-environments-using-anaconda) or [this guide](https://conda.io/docs/user-guide/tasks/manage-python.html).

**Comment by sarahefcarroll** [comment]
using Python 2.7 seems to be working, except of course now I get this error (see below).

I know these are probably really basic questions with obvious solutions, but I haven't actually used python to do more than graph a .csv file with an IR spectrum in years, and I really appreciate the help. 
![capture](https://user-images.githubusercontent.com/32676928/31402637-99c56c24-adc4-11e7-8c18-0953f2622fac.JPG)

**Comment by richford** [comment]
Good catch. Thanks for submitting an issue for that. I just pushed a new commit that should fix this. Could you pull the latest from this repo and try again?
